build/nodejs/build.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/bash
+#
+# CDDL HEADER START
+#
+# The contents of this file are subject to the terms of the
+# Common Development and Distribution License, Version 1.0 only
+# (the "License").  You may not use this file except in compliance
+# with the License.
+#
+# You can obtain a copy of the license at usr/src/OPENSOLARIS.LICENSE
+# or http://www.opensolaris.org/os/licensing.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+#
+# When distributing Covered Code, include this CDDL HEADER in each
+# file and include the License file at usr/src/OPENSOLARIS.LICENSE.
+# If applicable, add the following below this CDDL HEADER, with the
+# fields enclosed by brackets "[]" replaced with your own identifying
+# information: Portions Copyright [yyyy] [name of copyright owner]
+#
+# CDDL HEADER END
+#
+#
+# Copyright 2018 OmniTI Computer Consulting, Inc.  All rights reserved.
+# Use is subject to license terms.
+#
+# Load support functions
+. ../../lib/functions.sh
+PROG=node     # App name
+VER=v8.11.2      # App version
+VERHUMAN=$VER   # Human-readable version
+#PVER=          # Branch (set in config.sh, override here if needed)
+PKG=developer/nodejs            # Package name (e.g. library/foo)
+SUMMARY="Node.js is an evented I/O framework for the V8 JavaScript engine." # One-liner, must be filled in
+DESC="Node.js is an evented I/O framework for the V8 JavaScript engine.
+It is intended for writing scalable network programs such as web servers. " # Longer description, must be filled in
+
+BUILDARCH=64
+OPREFIX=$PREFIX
+PREFIX+="/$PROG-$VER"
```

Comment:
I would suggest to use a 2 digit version ... so that it ends up in 

`PREFIX+=/${PROG}-8.11`

the patch releases can then be treated as updates to this package while 8.12 would be a new package ...

Reply:
nginx is a good template for this, it includes:

```
MAJVER=${VER%.*}            # M.m
sMAJVER=${MAJVER//./}       # Mm
PREFIX+=/$PROG-$MAJVER
```